src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
> I've based my test on the fact, that last opened window can be accessed after reset method call, but that only worked for Selenium, but not Zombie. Hence the problem I'm facing.

and this means that you wrote a test which cannot reflect an actual issue for someone using the driver, as this is an unsupported usage.
So please write a test reproducing a real issue affecting the user (and if it is impossible to write one, there is no issue to be fixed)